SYCL/InvokeSimd/regression/call_vadd_1d_spill.cpp

```diff
@@ -0,0 +1,230 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+//
+// VISALTO enable run
+// RUN: env IGC_VISALTO=63 IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+
+// Tests invoke_simd support in the compiler/headers
+/* This program is basically an extension of the standard vector addition
+ * program (i.e., call_vadd_1d). The only difference is that instead of adding 2
+ * vectors (i.e., va + vb), we add 14 vectors. However, in this implemenation,
+ * the extra vectors beyond va and vb are zero vectors, so adding them has no
```

Comment:
Would it be difficult to make these extra vectors holding some meaningful non-zero values?
My concern here is that even if part of code or params/operands is lost or corrupted, then the test will not catch that.

Reply:
There are actually 2 cases in the test: one with zero extra vectors and one with non-zero. I fixed comment to represent real test configuration. Thanks for pointing out.